docs/install/install.rst

```diff
@@ -0,0 +1,237 @@
+Linux Compiling and Installing
+==============================
+
+This page contains sections as follows:
+
+.. contents::
+    :local:
+
+General Prerequisites
+*********************
+
+Freeciv21 has a number of prerequisites.  Note, that apart from the first prerequisite, the Freeciv21
+configuration process is smart enough to work out whether your system is suitable. If in doubt, just try it.
+
+* An operating system that support Qt.
+
+  Any modern operating system that support Qt is required. As of this writing this is Linux, Microsoft
+  Windows\ |reg| and Apple Mac OS X\ |reg|. On Windows Msys2 is supported.
+
+* A C and C++ 14 compiler.
+
+  Freeciv21 is written in very portable C and C++. Both 32- and 64-bit machines are supported. You cannot
+  use a "K&R C" compiler, or a C++ compiler. The C++ compiler must support C++ 14.

```

Comment:
C++ 17 is required in the build system, I probably failed to update the `README` when changing it but it's a good opportunity to fix.

Reply:
Corrected in next commit